WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -1,11 +1,13 @@
+// swiftlint:disable next opening_brace
 import ScreenObject
 import XCTest
 
-public class ProductsScreen: ScreenObject {
+public final class ProductsScreen: ScreenObject {
```

Comment:
Could you please provide more context about this change?

Reply:
Yup should've added a comment for this. Noticed that this was something that was missed in the last commit, all other screen classes are `final` classes so added this to match others.

Reply:
Thanks! I missed this. It makes sense for the screen objects to be `final`, IMO.